pkg/application/inject/fuse/injector.go

```diff
@@ -144,221 +149,175 @@ func (s *Injector) inject(in runtime.Object, pvcName string, runtimeInfo base.Ru
 	}
 	kind := typeMeta.Kind
 	log.V(1).Info("Inject application", "namespacedName", namespacedName, "kind", kind)
-
 	pods, err := application.GetPodSpecs()
 	if err != nil {
 		return
 	}
 
 	for _, pod := range pods {
-		metaObj, err := pod.GetMetaObject()
+		shouldSkipInjection, err := s.shouldSkipInjection(pod, namespacedName)
```

Comment:
I suggest naming the method name as `shouldInjection`.

Reply:
renamedit as `shouldInject`